scripts/binaryninja/README.md

```diff
@@ -0,0 +1,18 @@
+To use a plugin:
+
+1. Symlink the desired plugin into the directory specified here: https://github.com/Vector35/binaryninja-api/tree/dev/python/examples#loading-plugins
+2. `import` it from the Binary Ninja Script Console, and call desired functions
+
+Example:
+
+```
+$ ln -s $PWD/viz.py ~/Library/Application\ Support/Binary\ Ninja/plugins/
+
+```
+
+From script console:
+
+```
+>>> import viz
+>>> viz.viz(bv, '/mnt/hgfs/code/manticore/examples/linux/mcore_1vCAKM')
```

Comment:
mcore_ is a file that is generated by manticore but is not in the examples/ by default right?
It would be super helpful if we include an actual example_dir for this script particularly,
and also a screenshot of the plugin?